backend/api/Controllers/RobotController.cs

```diff
@@ -96,4 +97,43 @@ public async Task<ActionResult<Robot>> PostRobot([FromBody] Robot robot)
             throw;
         }
     }
+
+    /// <summary>
+    /// Start a mission for a given robot
+    /// </summary>
+    /// <remarks>
+    /// <para> This query starts a mission for a given robot and creates a report </para>
+    /// </remarks>
+    /// <returns> Report </returns>
+    [HttpPost]
+    [Route("{robotId}/start/{missionId}")]
+    [ProducesResponseType(typeof(Report), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
```

Comment:
Might not be necessary to include 500 as a response?

Reply:
I think 500 is always a possibility, if something unexpected happens. Or what do you think?

Reply:
I think 500 is always a possibility, if something unexpected happens. Or what do you think?

Reply:
Yes, it is but is is normal to document the "Some unknown error happened"?

Reply:
Hmm, not sure tbh. I dont feel we are documenting it so much, mot telling the user that it is a possibility. But I can remove it if we feel like it is unecessary.

Reply:
Keep it then, it's a small thing :D